src/daria/corrections/shape/curvature.py

```diff
@@ -423,7 +426,7 @@ def __call__(
 
         # Read image
         if isinstance(img, str) or isinstance(img, Path):
-            img = np.array(PIL_Image.open(Path(img)))
+            img = cv2.imread(img, cv2.IMREAD_UNCHANGED)
```

Comment:
```suggestion
            img = cv2.imread(str(img), cv2.IMREAD_UNCHANGED)
```
cv2 does not accept `Path` unfortunately. Again BGR -> RGB?